src/Mhwd.cpp

```diff
@@ -564,21 +565,25 @@ MHWD::STATUS Mhwd::uninstallConfig(Config *config)
     }
     else
     {
-        // TODO: Should we check for local requirements here?
-
-        // Run script
-        if (!runScript(installedConfig, MHWD::TRANSACTIONTYPE::REMOVE))
+        // Should we check for local requirements here?
+        if(!localRequirements.empty())
```

Comment:
Local requirements are checked before this function is called. We don't need that check here.